lib/config/schema.js

```diff
@@ -590,6 +590,12 @@ const CONFIG_SCHEMA = [
     defaultValue: [],
     deps: ['ignoreUserAgents'],
   },
+  {
+    name: 'apmClientHeaders',
+    configType: 'stringKeyValuePairs',
+    defaultValue: undefined,
+    envVar: 'ELASTIC_APM_APM_CLIENT_HEADERS',
+  },
```

Comment:
BTW, kudos to your config refactoring so far, David. Having this change be localized to just this is wonderful.